common.gypi

```diff
@@ -369,6 +370,32 @@
           }],
         ],
       }],
+      ['ubsan == 1 and OS != "mac" and OS != "zos"', {
+        'cflags+': [
+          '-fno-omit-frame-pointer',
+          '-fsanitize=undefined',
+        ],
+        'defines': [ 'UNDEFINED_SANITIZER'],
+        'cflags!': [ '-fomit-frame-pointer' ],
+        'ldflags': [ '-fsanitize=undefined' ],
+      }],
+      ['ubsan == 1 and OS == "mac"', {
+        'xcode_settings': {
+          'OTHER_CFLAGS+': [
+            '-fno-omit-frame-pointer',
+            '-gline-tables-only',
```

Comment:
For my curiosity: what is this for? Just a size saving measure?

Reply:
Yes, pretty much. But, honestly, I don't think this would benefit this specific workflow. I will remove it.